src/components/scenes/OtpErrorScene.tsx

```diff
@@ -180,7 +180,7 @@ export function OtpErrorScene(props: Props) {
       .finally(() => (inModal.current = false))
   })
 
-  const handleQrModal = useHandler(() => {
```

Comment:
Add an `await` to the `Airship.show` a few lines down. You can also get rid of the `.catch`, because the await will handle that.